lib/transaction/transaction.js

```diff
@@ -414,6 +414,11 @@ Transaction.prototype.lockUntilDate = function(time) {
   if (_.isDate(time)) {
     time = time.getTime() / 1000;
   }
+
+    for (var i = 0; i < this.inputs.length; i++) {
```

Comment:
A few things:
- This may not serialize because I think the property for the sequence number on an input is  `.sequenceNumber`. 
- There should be tests to cover these statements
- Indentation is incorrect